IdentityCore/src/util/mac/MSIDKeychainUtil.m

```diff
@@ -75,6 +75,13 @@ - (NSString *)getTeamId
             MSID_LOG_WITH_CTX(MSIDLogLevelError, nil, @"Failed to retrieve team identifier. Using bundle Identifier instead.");
             NSString *bundleIdentifier = [signingDic objectForKey:(__bridge NSString*)kSecCodeInfoIdentifier];
```

Comment:
perhaps just reading bundle Id differently would be a better workaround?

Reply:
Can you be more specific please.

Reply:
There are various ways to get a dynamic identifier for the app. For example, what if you try:

`[[NSBundle mainBundle] bundleIdentifier]`

Reply:
There may be no bundle. If memory serves, [NSBundle mainBundle] returns nil if the consuming app is a console tool. So there should be some fallback for this case anyway.

Reply:
This could work for us. Let me give that a try.